lib/muledump/mule.js

```diff
@@ -2066,8 +2066,11 @@
         var potions = d.Account.Potions;
         var items = ( typeof potions === 'string' ) ? potions.split(',') : undefined;
         if ( Array.isArray(items) === true && this.totals.cache === false ) {
-            for (var tp = 0; tp < items.length; tp++)
-                this.totals.count(items[tp], 'potions');
+            for (var tp = 0; tp < items.length; tp++) {
+                if (items[tp] !== "-1") {
```

Comment:
I'm not counting empty potion storage as an empty slot in totals.
If that's desired, the Greater potion fix has to be applied here too.